Web_Development/LightFlow/lightflow/plugins/shell_task.py

```
@@ -0,0 +1,106 @@
+"""
+Shell Task Plugin - Executes shell commands as tasks.
+"""
+
+import subprocess
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)